theorems/T000364.md

```diff
@@ -0,0 +1,14 @@
+---
+uid: T000364
+if:
+  P000128: true
+then:
+  P000018: true
+refs:
+- doi: 10.1016/j.topol.2005.07.015
+  name: Applications of k-covers II
+---
+
+Mentioned in the proof of Proposition 5 in {{doi:10.1016/j.topol.2005.07.015}}.
+
+Given any cover $\mathscr U$ of a $k$-Lindelöf space $X$, consider $\mathscr V = \left\{ \bigcup \mathscr F : \mathscr F \in [\mathscr U]^{<\omega} \right\}$ where $[\mathscr U]^{<\omega}$ denotes the set of finite subsets of $\mathscr U$. Then $\mathscr V$ is a $k$-cover of $X$ and we can find a countable subset which is a $k$-cover of $X$. Notice that such a subset corresponds to a countable subset of $\mathscr U$ which is a cover of $X$.
```

Comment:
"Given any cover" --> "Given any open cover".
At the end, possibly "corresponds to a countable subset of $\mathscr U$ which is a cover of $X$." --> "corresponds to a countable subcover of $\mathscr U$"

Reply:
Changed.